leetcode/generate_md.py

```diff
@@ -1,63 +0,0 @@
-separator = '<!-----solution----->'
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 Add files via upload 
 Mistakes: 
 Where have you found such homework(Add)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(files), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do(via)? It is not from ['Added', 'Deleted'].